docs/data/sdks/ampli-overview.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Ampli
+description: Overview to Amplitude Analytics SDK architecture and common usage
+---
+
+## Overview
+
+**Ampli** dynamically generates a light-weight wrapper for the **Amplitude SDK** based on your analytics tracking plan in **Amplitude Data** making event tracking easier and less error prone.
+
+This document provides a high-level overview of Ampli. A [video demo](https://academy.amplitude.com/instrumenting-events-with-amplitude-data-and-the-ampli-cli) is also available in Amplitude Academy.
+
+<figure markdown>
+  ![How Ampli fits into your workflow](../../assets/images/ampli/overview/ampli-overview.drawio.svg)
+  <figcaption>How Ampli fits into your workflow</figcaption>
+</figure>
+
+The **Ampli Wrapper** provides types and methods that prevent human error by strictly enforcing event names and property values. The wrapper code enables autocompletion for all events and properties in your tracking plan, as well as static type checks at development and compile time.
```

Comment:
I avoided subsections here since this is all part of the overview and is mostly a short list of benefits.

Users can navigate to the existing sections (Amplitude Data, Ampli CLI, and Ampli Wrapper) for details on that area.